group_vars/galaxyservers.yml

```diff
@@ -8,7 +8,7 @@ galaxy_manage_cleanup: true # Requires tmpwatch(8) or tmpreaper(8) present
 galaxy_layout: root-dir
 galaxy_root: /srv/galaxy
 galaxy_user: {name: "{{ galaxy_user_name }}", shell: /bin/bash}
-galaxy_commit_id: release_24.1
+galaxy_commit_id: 7e360e2c17c06048198bb2fe4c23ce3403509a69
```

Comment:
you are changing the default for all hosts (and this one gets overiden later in host_vars), you need to change it in https://github.com/CESNET/usegalaxy/blob/main/host_vars/galaxy-qa2.galaxy.cloud.e-infra.cz/vars.yml